### PR TITLE
add taskgroup 0.0.0a4

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

References:
- blocks https://github.com/conda-forge/hypercorn-feedstock/pull/44

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/help-python ready for review

File: recipes/taskgroup/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.0a4" %}
```
Comment (ocefpaf) [inline recipes/taskgroup/meta.yaml:1]: This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Comment (bollwyvl) [reply recipes/taskgroup/meta.yaml:1]: Hm, [no stable release](https://pypi.org/project/taskgroup/#history) exists, and this is [depended upon](https://github.com/conda-forge/hypercorn-feedstock/pull/44) by an existing feedstock.

Comment (ocefpaf) [reply recipes/taskgroup/meta.yaml:1]: You can place this one is a dev channel and still use it there by adding an extra channel in the conda-forge yaml file. However, that is too cumbersome and, IMO, people should just stop with the dev releases that should not have been a dev release to start with. PyPI set a bad precedent with that "pre" policy IMO.

Comment (bollwyvl) [reply recipes/taskgroup/meta.yaml:1]: I heartily agree, and really `black` did the worst damage, leaving a lot of people with `--pre` on in various setups. Indeed, having a second `dev` registry, and the _ability_ to specify it per dep would have likely have been more useful, as quietly allowing a `==` pin a pre-release package feels super bad.

Comment (ocefpaf) [reply recipes/taskgroup/meta.yaml:1]: And black did not need that BTW! It was the only pre release that could be installed as a stable one. Go figure why...

**Pull request merged**